gcloud-java-logging/src/main/java/com/google/cloud/logging/spi/DefaultLoggingRpc.java

```diff
@@ -94,10 +90,21 @@ private InternalLoggingOptions(LoggingOptions options) {
     protected ExecutorFactory<ScheduledExecutorService> executorFactory() {
       return super.executorFactory();
     }
+
+    @Override
+    protected ApiCallSettings.Builder apiCallSettings() {
```

Comment:
Methods are protected and here we are in a different package. We do this to avoid exposing these methods to the user.